### DISCUSSION TITLE
sms and chat interfaces.

### DISCUSSION THREAD

**Discussion by markdav-is** [OpenEugene/fire-refuge#10] [Ideas]
it would be nice to let folks connect using test messages.  It would also be nice to connect folks to services using just a phone number.  

one use case would be, visit the site an fill out a form for 2 adults and 1 child for the night.  The location and need get posted to the server, we find a match to a site with capacity and then send an SMS with the number of the shelter.

another use case is verifying inventory.  we can send a txt to the number for the shelter and ask them for an inventory update.  we can accept that and update the inventory level.

**Comment by thomasfiorelli** [comment]
This is certainly possible. Here is the integration tool I have used to send SMS to survey participants. https://www.integromat.com/en/mobile
